.vuepress/components/HighlightBox.vue

```diff
@@ -84,13 +84,8 @@
         }
 
         &.warn, &.warning {
-            background: var(--color-warning);
-            color: black
-            fill: black
-
-            .title {
-                color: black
-            }
+            background: linear-gradient(78.06deg, #00B067 0.2%, #FFD303 97.95%);
```

Comment:
Maybe this is splitting hair:

1. A value like `78.06deg` makes it look like it was created approximatively. Does it matter if it was `78deg`?
2. Line 77 we have `98.88%` as the last value, and here we have `97.95%`. Were they intended to be equal but you set them with a slider instead of precise values?
3. Not sure if it makes sense in this case, but it could be `const gradientAngle = "78 deg";` and reuse it. Same for other repeated values.